src/Relude/Extra/Validation.hs

```diff
@@ -155,17 +155,19 @@ instance Semigroup e => Applicative (Validation e) where
     {-# INLINE (<*>) #-}
 
     (*>) :: Validation e a -> Validation e b -> Validation e b
-    Failure el *> Failure er = Failure (el <> er)
-    Failure e  *> Success _  = Failure e
-    Success _  *> Failure e  = Failure e
-    Success _  *> Success b  = Success b
+    Failure e *> b = Failure $ case b of
+                                 Failure e' -> e <> e'
```

Comment:
My understanding would be that implementing operators `*>` and `<*` manually is unnecessary. They both implemented in terms of `<*>` which already shortcuts early. Implementing them manually could be error-prone in the case of this instance and performance benefits are not obvious. So I think it's better to just remove the implementations for `*>` and `<*`

Reply:
Isn't it slightly faster to have all of these operators explicitly written, at least in the case where both sides are `Success`?

Reply:
It _may be_ slightly faster. But GHC is very good at optimizing. And it's hard to predict how faster without benchmarking or looking at the core. On the other hand, the cost of the possible error and maintaining more code is higher. So, if we going to change code for `Validation`, it's better to wait for issue #176 to be implemented first to make sure that the refactoring is indeed valid.